config/formats.js

```diff
@@ -397,11 +397,35 @@ exports.Formats = [
 			this.add("raw|<h1>BATTLE!</h1>");
 		},
 	},
-
+	{
+		name: "Digimon Showdown",
+		section: "Exiled's Custom Gamemodes",
+		mod: "digimon",
+		team: "randomDigimon",
+		gameType: "doubles",
+		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		onBegin: function () {
+			this.add('c|~Insist|Hey players, I\'m the Head Developer of this gamemode, and for some reason I\'m here to give you a **DISCLAIMER**, so be ready....');
+			this.add('c|~Insist|This gamemode was made solely for the enjoyment of the players of the Pokemon Showdown community, and we do not claim to own or be responsible to any rights/copyrights related to Digimon.');
+			this.add('c|~Insist|Now that that\'s out of the way, go ahead and meet our developers! Oh, wow! Here they are, let them introduce themselves.');
+			this.add('c|%Ashley the Pikachu|Haha, hey! I am the Head Researcher that made this gamemode possible!');
+			this.add('c|%Ashley the Pikachu|Oh yeah, if you need some information on this meta I made a manual for a few basic things, check it out, it\'s right below this.');
+			this.add('c|+Stellation|Hey, I was in charge of assisting the other developers. I did my own fair share of coding when needed and made this thing open source so you all can download, edit, and play this meta! :3');
+			this.add('c| AlfaStorm|Hey, you see all those items, and move animations, haha that was my job ^~^');
+			this.add('c|~HoeenHero|Hey, I helped creating mechanics for this gamemode to work as it does :D');
```

Comment:
It would probably be best to include the credits in the format description instead of posting messages at the start of every match, also you dont need to include me.